randovania/data/json_data/prime2.txt

```diff
@@ -9854,6 +9861,9 @@ Asset id: 1815493844
           All of the following:
               Space Jump Boots
               Combat/Scan Dash (Intermediate) or Standable Terrain (Intermediate)
+          Morph Ball and Spider Ball and Morph Ball Bomb and Bomb Jump (Intermediate)
+  > Out of Bounds
+      Single Room Out of Bounds (Expert) and Use Screw Attack (No Space Jump)
```

Comment:
Is the NoSJ SA without Screw Attack at Z-Axis intended?

Reply:
That's below the zaxis, so it's unaffected